gradle/libs.versions.toml

```diff
@@ -160,8 +160,8 @@ spark-sql-v32-v212 = { module = "org.apache.spark:spark-sql_2_12", version = { s
 spark-sql-v32-v213 = { module = "org.apache.spark:spark-sql_2_13", version = { strictly = "[3.2, 3.3[", prefer = "3.2.4"}}
 spark-sql-v33-v212 = { module = "org.apache.spark:spark-sql_2_12", version = { strictly = "[3.3, 3.4[", prefer = "3.3.3"}}
```

Comment:
Can we also bump 3.3.3 to 3.3.4 ? 
https://spark.apache.org/downloads.html

Reply:
Also we might have to remove spark-sql-v32-v212 and spark-sql-v32-v213

Reply:
> Can we also bump 3.3.3 to 3.3.4 ? https://spark.apache.org/downloads.html

3.3 is not part of this PR

Reply:
> Also we might have to remove spark-sql-v32-v212 and spark-sql-v32-v213

Similar - out of scope of this PR - there is already another one.